_targets.R

```diff
@@ -1,38 +1,83 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
+source("1_fetch/src/download_nwis_site_data.R")
+source("1_fetch/src/nwis_site_info.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+# Global parameters
+parameterCd <- '00010'
+startDate <- "2014-05-01" 
+endDate <- "2015-05-01"
+
+
+# For each site, download data and write to csv
+#         Note: this feels inefficient and hard to scale up. How can we "loop" over
+#               multiple sites without making a target specified to each individually?
 p1_targets_list <- list(
   tar_target(
+    site_data_01427207, 
+    download_nwis_site_data(site_num = "01427207",
+                            parameterCd = parameterCd,
+                            startDate = startDate,
+                            endDate = endDate,
+                            out_dir = "1_fetch/out/"),
+  ),
+  tar_target(
+    site_data_01432160,
+    download_nwis_site_data(site_num = "01432160",
+                            parameterCd = parameterCd,
+                            startDate = startDate,
+                            endDate = endDate,
+                            out_dir = "1_fetch/out/"),
+  ),
+  tar_target(
+    site_data_01436690,
+    download_nwis_site_data(site_num = "01436690",
+                            parameterCd = parameterCd,
+                            startDate = startDate,
+                            endDate = endDate,
+                            out_dir = "1_fetch/out/"),
+  ),
+  tar_target(
+    site_data_01466500,
+    download_nwis_site_data(site_num = "01466500",
+                            parameterCd = parameterCd,
+                            startDate = startDate,
+                            endDate = endDate,
+                            out_dir = "1_fetch/out/"),
+  ),
+  tar_target(
+    # Merge data from all sites
     site_data,
-    download_nwis_data(),
+    rbind(site_data_01427207,
```

Comment:
The `site_data_*` targets are currently straddling being a file target or an object target and need to be just one or the other. Once they are fully file targets (meaning they return the filepath), you need to adjust this target to first read in the files and then bind them. 

We try to use `tidyverse` functions as much as possible, so here you could use `bind_rows()` instead of `rbind()` which is just a bit more robust.

Here's one way to do this:

```r
lapply(c(site_data_01427207, site_data_01436690, ...), read_csv) %>% bind_rows()
```